chore(doc): include policy for joining teams (#525)
nodejs/admin: GITHUB_ORG_MANAGEMENT_POLICY.md

In certain cases, OpenJS Cross Project Council and/or OpenJS Foundation Board
of Directors approval may also be required.

### Repository Team Ownership

When making a request to create a new repository, specify the team(s) that will
have write or admin access. If there is not an appropriate team to maintain a
new repository, request a new team. Approval is automatic if there are no
objections from TSC or CommComm after 72 hours.

## Teams

To join a Working Group team or a team who fall under the charter of a Working

> Group, read that Working Group governance documentation. For other teams, send

Reviewer: ```suggestion
Group, read that Working Group's governance documentation. For other teams, send
```

> an issue on the [Node.js admin repository][nodejs/admin].

Reviewer 2: This isn't how CommComm does things afaik - generally members are PR-ed in and then the Initiatives (which is a group set left out here) request that the only member of CommComm who is an org owner (the chairperson) add them 😬

Author: That's also how most working groups work AFAIK. The policy/guidelines proposed here should only apply to teams who are not tied to a repository.

Reviewer 2: Ah my bad. Failure of my reading. Would it be possible to clarify that in more specific language in `For other teams, [...]` ?

Additionally, it would be nice if `To join a Working Group team or a team` also included `Initiatives`.

the Node.js org, open an issue on the [Node.js admin repository][nodejs/admin].

For members of teams with org-wide admin permissions (TSC, CommComm and
Moderation), it is not possible to send a join request to any teams, since
those users have permission to add themselves to the team. In this case, use
your best judgement to decide between adding yourself to the team or asking for
permission. When joining a `*-admin` which is not bound to a working group or

> repository, it is suggested to open an issue on nodejs/admin pinging the team,

Reviewer: ```suggestion
repository, open an issue in the nodejs/admin repo pinging the team,
```

> start a discussion on the team, or contact the team maintainers directly to ask

Reviewer: ```suggestion
inquire using the team discussion feature, or contact the team maintainers directly to ask
```
